pkgs/applications/graphics/monado/default.nix

```
@@ -46,25 +45,16 @@
 
 stdenv.mkDerivation rec {
   pname = "monado";
-  version = "21.0.0";
+  version = "21.0.0+date=2022-05-21";
 
   src = fetchFromGitLab {
     domain = "gitlab.freedesktop.org";
     owner = pname;
     repo = pname;
```

```suggestion
    owner = "monado";
    repo = "monado";
```
makes it easier to get to the source url.